granulate_utils/linux/proc_events.py

```diff
@@ -116,7 +124,14 @@ def _listener_loop(self):
                 break
 
             for key, _ in events:
-                data = key.fileobj.recv(256)
+                try:
+                    # When stressed, reading from the socket can raise
+                    #   OSError: [Errno 105] No buffer space available
+                    # This seems to be safe to ignore, empirically no events were missed
```

Comment:
Interesting. Did you see it when there was a large amount of processes or on general network stress?

Reply:
Large amount of processes. I stressed it on purpose locally

Reply:
Hmm maybe I'll add packet filtering at some point which might solve this